v1/StorageSpec.dhall

```diff
@@ -4,10 +4,39 @@ let Kubernetes = imports.Kubernetes
 
 let EmbeddedPersistentVolumeClaim = ./EmbeddedPersistentVolumeClaim.dhall
 
+let Common = { disableMountSubPath : Optional Bool }
+
+let EmptyDir = Common ⩓ { emptyDir : Kubernetes.EmptyDirVolumeSource.Type }
+
+let VolumeClaimTemplate =
+      Common ⩓ { volumeClaimTemplate : EmbeddedPersistentVolumeClaim.Type }
+
+let common = { disableMountSubPath = None Bool }
+
 let StorageSpec =
-      < EmptyDir : { emptyDir : Kubernetes.EmptyDirVolumeSource.Type }
-      | VolumeClaimTemplate :
-          { volumeClaimTemplate : EmbeddedPersistentVolumeClaim.Type }
-      >
+      { Union =
```

Comment:
If you're going to make this change then I prefer that it be called `Type` not `Union`